src/app/Exceptions/BackpackProRequiredException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Backpack\CRUD\app\Exceptions;
+
+class BackpackProRequiredException extends \Exception
+{
+    /**
+     * Render the exception into an HTTP response.
+     *
+     * @param  \Illuminate\Http\Request
+     * @return \Illuminate\Http\Response
+     */
+    public function render($request)
+    {
+        // 0, by default, pass only what is a feature we construct the rest of the message
+        // 1 use the provided message in full
+        switch ($this->getCode()) {
+            case 0:
+                $this->message = $this->message.' is a Backpack PRO feature. Please purchase and install <a href="https://backpackforlaravel.com/pricing">Backpack\PRO</a>.';
```

Comment:
![tv4-problem-tom-hanks-houston-we-have-a-4Hx5nJBfi8FzFWxztb](https://user-images.githubusercontent.com/1032474/163144309-1a6fa7b3-74f5-43f8-ab8b-0fedbe661e81.gif)

The lastest error views output `e($exception->getMessage())` so the URL won't work...

Reply:
So let's remove the URL entirely, in favor of something like:
```suggestion
                $this->message = $this->message.' is a Backpack PRO feature. Please purchase and install the Backpack\PRO addon from backpackforlaravel.com';
```